.travis.yml

```diff
@@ -64,7 +64,7 @@ before_script:
 
   # report some good version numbers to the build
   - gcc --version
-  - (! var_search "${TRAVIS_SDK-}" elf || arm-none-eabi-gcc --version)
+  - (! var_search "${TRAVIS_SDK-}" arm || arm-none-eabi-gcc --version)
```

Comment:
Ooops! thanks for spotting this.  Could be its own PR, if the update to enial is held back for any other reason.